colcon-build-and-test/action.yaml

```diff
@@ -72,7 +72,7 @@ runs:
         colcon test --event-handlers console_cohesion+ \
           --packages-above ${{ inputs.target-packages }} \
           --return-code-on-test-failure
-        colcon lcov-result --packages-above ${{ inputs.target-packages }}
+        (find build -name coverage.info | grep .) && colcon lcov-result --packages-above ${{ inputs.target-packages }}
```

Comment:
What's `grep .` for?

Reply:
I forgot to add the reference. I followed this:
https://serverfault.com/questions/225798/can-i-make-find-return-non-0-when-no-matching-files-are-found/768042

Reply:
How about using `test -f` if the path is always the same?
https://github.com/autowarefoundation/autoware-github-actions/pull/30/files#r792285770

Reply:
I tried test command but it cannot use wildcard.
`test -f "*/coverage.info"`

Reply:
Could you show me the exact path where `coverage.info` is located?

Reply:
Sure. I copied console log of my local repository as an example:

```
autoware.universe$ find . -name coverage.info
./build/autoware_auto_geometry_msgs/coverage.info
./build/autoware_auto_mapping_msgs/coverage.info
./build/autoware_lint_common/coverage.info
./build/ndt_pcl_modified/coverage.info
```

Reply:
Thank you, how about this?

```
if [ ! -z $(find build/ -name coverage.info) ];
```

Reply:
Looks good! a975c35